src/Context/ContextDefinition.php

```diff
@@ -81,7 +81,12 @@ public static function createFromArray($values) {
     // Default to Rules context definition class.
     $values['class'] = isset($values['class']) ? $values['class'] : '\Drupal\rules\Context\ContextDefinition';
     if (!isset($values['type'])) {
-      $values['type'] = 'any';
+      if (!isset($values['value'])) {
+        $values['type'] = 'any';
+      }
+      else {
+        $values['type'] = $values['value'];
+      }
```

Comment:
why do we have to do this? Can't we just change $nameMap and point 'value' => 'dataType'? Or are we using 'type' as key anywhere else?